common-npm-packages/azure-arm-rest-v2/package.json

```diff
@@ -14,15 +14,15 @@
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
     "jsonwebtoken": "7.3.0",
-    "q": "1.4.1",
+    "q": "1.5.1",
     "typed-rest-client": "1.7.3",
-    "azure-pipelines-task-lib": "2.11.2",
-    "@types/mocha": "2.2.48",
-    "@types/node": "6.0.68",
-    "@types/q": "1.0.7"
+    "azure-pipelines-task-lib": "^3.0.6-preview.0",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^10.17.0",
+    "@types/q": "1.5.4"
   },
   "devDependencies": {
-    "typescript": "2.3.4",
+    "typescript": "4.0.2",
     "shelljs": "0.8.4"
```

Comment:
Tasks can only have typescript in devDependencies, does the same limitation apply to common npm packages?

Reply:
I think yes.